scripts/lint-test-window-construction.py

```
@@ -0,0 +1,216 @@
+    """Object files that reference the designated initializer.
+
+    The sanctioned factory is expected to; nothing else in the test target should.
+    """
+    offenders = []
+    sanctioned_stem = Path(SANCTIONED).stem
+    for obj in sorted(objects_dir.rglob("*.o")):
+        if obj.stem == sanctioned_stem:
+            continue
+        # Read bytes: an object file's string table is not valid UTF-8, and decoding it throws.
+        try:
+            dump = subprocess.run(
```

Ruff (0.15.21)

[error] `subprocess` call: check for execution of untrusted input

(S603)